website/docs/language/import/index.mdx

```diff
@@ -0,0 +1,123 @@
+---
+page_title: Import - Configuration Language
+description: >-
+  Import and manage existing resources with Terraform using configuration-driven import.
+---
+
+# Import
+
+-> **Note:** Import blocks are only available in Terraform v1.5.0 and later.
+
+~> **Experimental:** Import blocks are an experimental Terraform feature. While we do not expect to make backwards-incompatible changes to syntax, the processing of imports during the plan stage, generating configuration, and the `-generate-config-out` flag may change in future releases.
```

Comment:
> Import blocks are an experimental Terraform feature

Did you mean to state this? or did you mean to only state that config gen is experimental?